src/main/java/org/gridsuite/ds/server/service/parameters/impl/ParametersServiceImpl.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.ds.server.service.parameters.impl;
+
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.dynamicsimulation.DynamicSimulationParameters;
+import com.powsybl.dynamicsimulation.json.JsonDynamicSimulationParameters;
+import com.powsybl.dynawaltz.DynaWaltzParameters;
+import org.gridsuite.ds.server.service.parameters.ParametersService;
+import org.gridsuite.ds.server.utils.xml.XmlMerge;
+import org.gridsuite.ds.server.utils.xml.implementation.SimpleXmlMerge;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@Service
+public class ParametersServiceImpl implements ParametersService {
+
+    @Override
+    public byte[] getEventModel() throws IOException {
+        // read the events.groovy in the "parameters" resources
+        return getClass().getResourceAsStream(Paths.get(PARAMETERS_DIR, EVENTS_GROOVY).toString()).readAllBytes();
+    }
+
+    @Override
+    public byte[] getCurveModel() throws IOException {
+        // read the curves.groovy in the "parameters" resources
+        return getClass().getResourceAsStream(Paths.get(PARAMETERS_DIR, CURVES_GROOVY).toString()).readAllBytes();
+    }
+
+    @Override
+    public DynamicSimulationParameters getDynamicSimulationParameters(byte[] dynamicParams) throws IOException, RuntimeException {
+
+        // prepare a temp dir for current running simulation
+        Path configDir = PlatformConfig.defaultConfig().getConfigDir().orElseThrow();
+        Path workingDir = Files.createTempDirectory(configDir, WORKING_DIR_PREFIX);
```

Comment:
This is not ideal to create a working temporary directory inside a config directory, but I do not see how to do otherwise if we want to keep file system abstraction.